### PR TITLE
Deprecate 'get' and 'function' keywords by printing a warning on every usage

### LINKED ISSUE #3: Deprecate "get" and "function" syntax
We can now declare sdkgen functions with `fn getSomething(a: int): Something`. But this can still be written as `get something(a: int): Something` or as `function getSomething(a: int): Something`. The last two should be deprecated.

A warning should be generated by the parser when they are used. But they should still work.

### PR DESCRIPTION
**Pull request by lbguilherme**
Fixes #3

### CODE REVIEW

File: cli/src/build.ts [hunk 1]
```diff
@@ -43,29 +43,29 @@ export function buildCmd(argv: string[]) {
             ]),
         );
         process.exit(0);
-        return;
```
Comment (dgadelha) [inline cli/src/build.ts]: Why was this removed?

Comment (lbguilherme) [reply cli/src/build.ts]: It wasn't reachable. Dead code
